docs/installation/getting-started/quick-start.md

```diff
@@ -4,318 +4,1720 @@
+					mode: 'manual',
+					label: 'Downloadable',
+					attributes: {
+						download: 'file'
+					}
+				}
+			}
+		},
+		mention: {
+			feeds: [
+				{
+					marker: '@',
+					feed: [
+						'@apple', '@bears', '@brownie', '@cake', '@cake', '@candy', '@canes', '@chocolate', '@cookie', '@cotton', '@cream',
+						'@cupcake', '@danish', '@donut', '@dragée', '@fruitcake', '@gingerbread', '@gummi', '@ice', '@jelly-o',
+						'@liquorice', '@macaroon', '@marzipan', '@oat', '@pie', '@plum', '@pudding', '@sesame', '@snaps', '@soufflé',
+						'@sugar', '@sweet', '@topping', '@wafer'
+					],
+					minimumCharacters: 1
+				}
+			]
+		},
+		placeholder: 'Type or paste your content here!',
+		table: {
+			contentToolbar: [
+				'tableColumn', 'tableRow', 'mergeTableCells', 'tableProperties', 'tableCellProperties', 'toggleTableCaption'
+			]
+		},
+		wproofreader: {
+		// PROVIDE CORRECT VALUE HERE:
+			serviceId: 'service ID',
+			lang: 'auto',
+			srcUrl: 'https://svc.webspellchecker.net/spellcheck31/wscbundle/wscbundle.js'
+		},
+		pagination: {
+			// A4
+			pageWidth: '21cm',
+			pageHeight: '29.7cm',
+
+			pageMargins: {
+				top: '20mm',
+				bottom: '20mm',
+				right: '12mm',
+				left: '12mm'
+			}
+		},
+		licenseKey: '', // PROVIDE THE CORRECT VALUE.
+		sidebar: {
+			container: document.querySelector( '.sidebar-container' )
+		},
+		revisionHistory: {
+			showRevisionViewerCallback: config => {
+				const editorContainer = document.querySelector( '#editor-container' );
+				const viewerContainer = document.querySelector( '#revision-viewer-container' );
+				const viewerElement = document.querySelector( '#revision-viewer-editor' );
+
+				config.revisionHistory.viewerSidebarContainer = document.querySelector( '#revision-viewer-sidebar' );
+
+				return CKEDITOR.DecoupledEditor.create( viewerElement, config ).then( viewerEditor => {
+					viewerContainer.style.display = 'flex';
+					editorContainer.style.display = 'none';
+
+					const toolbarContainer = document.querySelector( '.editor-toolbar' );
+					toolbarContainer.innerHTML = '';
+					toolbarContainer.appendChild( viewerEditor.ui.view.toolbar.element );
+
+					return viewerEditor;
+				} );
+			},
+			closeRevisionViewerCallback: viewerEditor => {
+				const editorContainer = document.querySelector( '#editor-container' );
+				const viewerContainer = document.querySelector( '#revision-viewer-container' );
+
+				viewerContainer.style.display = 'none';
+				editorContainer.style.display = '';
+
+				return viewerEditor.destroy().then( () => {
+					const toolbarContainer = document.querySelector( '.editor-toolbar' );
+					toolbarContainer.innerHTML = '';
+					toolbarContainer.appendChild( window.editor.ui.view.toolbar.element );
+				} );
+			}
+		},
+	} )
+	.then( editor => {
+		const toolbarContainer = document.querySelector( '.editor-toolbar' );
+
+		toolbarContainer.appendChild( editor.ui.view.toolbar.element );
+	} )
+	.catch( error => {
+		console.log( error );
+	} );
+```
```

Comment:
As above, the unnecessary huge amount of code and repetition of most of it just behind, I think we might delete this and refer to the main example explaining the details.

Reply:
@dufipl - please compare the solution we proposed with collapsible code listings.